File: OrangesAndChocolate/Models/Recipe.cs
namespace OrangesAndChocolateB.Models
{
    
    public class Recipe
    {
        public int ID { get; }
        public string Name { get; set; }
        public List<Ingredient> Ingredients { get; set; }
        public string Directions { get; set; }
        public bool isActive { get; set; }
        public string CreatorUsername { get; set; }

        public Recipe()
        {
            Ingredients = new List<Ingredient>();
        }

    }
}

File: frontend/OrangesAndChocolate/src/app/Models/Recipe.ts
export class Recipe{
    name: string;
    creatorUserame: string;
    ingredients: string[]
    directions: string;

    constructor(Name: string, CreatorName : string, Ingredients : string[], dir: string){
        this.name = Name;
        this.creatorUserame = CreatorName;
        this.ingredients = Ingredients;
        this.directions = dir;
    }
}